File: Assignment-37/Program4.java
import  java.util.*;

class Program4
{
      public static void main(String arg[])   
      {
            Pattern pobj = new Pattern();
            Scanner sobj = new Scanner(System.in);

            System.out.println("Enter numbers of rows :");
            int i = sobj.nextInt();

            System.out.println("Enter numbers of colum :");
            int j = sobj.nextInt();

            pobj.Display(i,j);
      }
}
class Pattern
{
      public void Display(int iRow ,int iCol)
      {
            for(int i = 1;i<iCol;i++)
            {
                  for(int j = 1;j<iRow;j++)
                  {

                        System.out.print("*\t#\t");
                        
                  }
                  System.out.println();
            }
      }
}

File: Assignment-20/program4.c
#include<stdio.h>

void Display(int iRow , int iCol)
{
      int i =0 ,j =0;

      for(i = iRow; i >= 1;i--)
      {
            for(j = 1;j <= iCol;j++)
            {
                  if((i==1)||(i==iRow))
                  {
                        printf("%d\t",i);
                  }
                  else if(i <= iRow)
                  {
                        printf("%d\t",i);
                  }
                 
            }

            printf("\n");
      }
}
int main()
{
      int ivalue1 , ivalue2 = 0;

      printf("Enter the number of Row :");
      scanf("%d",&ivalue1);

      printf("Enter The number of colum :");
      scanf("%d",&ivalue2);

      Display(ivalue1 ,ivalue2);

      return 0;
}